requirements.txt

```diff
@@ -1,12 +1,12 @@
 # START
 
 # from invenio/requirements.txt
-#
 -e git+git://github.com/mrjoes/flask-admin.git#egg=Flask-Admin-1.0.9.dev0
--e git+https://github.com/inveniosoftware/invenio-previewer-ispy.git#egg=invenio-previewer-ispy
+-e git+https://github.com/tpmccauley/invenio-previewer-ispy.git@webgl#egg=invenio-previewer-ispy
```

Comment:
it should be changed back when inveniosoftware/invenio-previewer-ispy#8 is merged